molecule/cookiecutter/molecule/{{cookiecutter.role_name}}/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/molecule.yml

```diff
@@ -44,7 +44,7 @@ platforms:
     flavor: NO-Nano
 {%- elif cookiecutter.driver_name == 'podman' %}
   - name: instance
-    image: centos:7
```

Comment:
rebase and adopt the configurable option what we are using with other tests, so we can switch images.